src/platform/lunarlake/include/platform/lib/shim.h

```diff
@@ -47,7 +47,8 @@
 #define L2HSBPM(x)			(0x17A800 + 0x0008 * (x))
 #define SHIM_HSPGCTL(x)			(L2HSBPM(x) + 0x0000)
 
-#define LSPGCTL				0x71D80
+#define LSPGCTL0		0x71D80
```

Comment:
@dnikodem can we read this from Zephyr headers today ?

Reply:
ok, I will update this PR, thanks.